packages/console/src/screens/app-builder/panels/explorer/Layers.tsx

```diff
@@ -6,7 +6,7 @@ import type { LayerOptions } from "./interfaces";
 import { LayerContextProvider } from "./layers/LayerContextProvider";
 import { LayerManagerProvider } from "./manager/LayerManagerProvider";
 
-export { useLayer } from "./layers";
+export { useLayer } from "./layers/index";
```

Comment:
This seems a little odd. Shouldn't TS/Webpack pickup `../layers/index.ts` via `../layers`?